Fortnox.NET.Tests/Fortnox.NET.Tests/InvoiceTests.cs

```diff
@@ -157,9 +158,11 @@ public async Task CreditInvoiceTest()
             Assert.IsTrue(bookkeeptInvoice.Booked);
 
             var creditedInvoice = await InvoiceService.CreditInvoiceAsync(request, response.DocumentNumber);
+            var updatedDebitInvoice = await InvoiceService.GetInvoiceAsync(request, response.DocumentNumber);
 
-            Assert.AreEqual("0", response.CreditInvoiceReference);
-            Assert.IsTrue(!string.IsNullOrEmpty(creditedInvoice.CreditInvoiceReference));
+            Assert.IsTrue(creditedInvoice.CreditInvoiceReference.HasValue);
+            Assert.IsTrue(updatedDebitInvoice.CreditInvoiceReference.HasValue);
```

Comment:
is there an actual expected value we can assert against, rather than just having a value?